test-apps/ui-test-app/package.json

```diff
@@ -107,6 +107,7 @@
     "@itwin/presentation-components": "workspace:*",
     "@itwin/presentation-frontend": "workspace:*",
     "@itwin/ui-items-providers-test": "workspace:*",
+    "extension-prototype": "workspace:*",
```

Comment:
until we add the extension loader in the ts files, id hold off on adding this dep

Reply:
Removed the dependency